changelog.txt

```diff
@@ -9,6 +16,7 @@
 * Fix   - Correct validation for UPS fields in Carrier Account connect form.
 * Tweak - Add message to explain automated tax requires tax-exclusive product pricing.
 * Fix   - Disable USPS refunds for untracked labels only.
+* Fix   - Adjust documentation links.
```

Comment:
Looks like I botched the changelog version [in my PR](https://github.com/Automattic/woocommerce-services/pull/2268/files), somehow 😬